src/ports/SkFontMgr_win_dw.cpp

```diff
@@ -85,7 +85,12 @@ SK_STDMETHODIMP_(ULONG) StreamFontFileLoader::AddRef() {
 SK_STDMETHODIMP_(ULONG) StreamFontFileLoader::Release() {
     ULONG newCount = InterlockedDecrement(&fRefCount);
     if (0 == newCount) {
-        delete this;
+        // AddRef and Release calls made by the system when replaying
+        // do not match up exactly with what happened when recording, which
+        // can lead to this object being prematurely freed when replaying.
+        // For now we workaround this by letting the loader leak when replaying.
+        if (!SkRecordReplayIsReplaying())
```

Comment:
Should we add the leak feature name?

Reply:
Features are intended to adjust behavior when recording, and this code only runs while replaying, so we don't need a feature test.